includes/class-sensei-assets.php

```diff
@@ -289,4 +289,17 @@ public function get_icon( string $name, string $class_names = '' ) : string {
 
 		return '<svg class="' . esc_attr( $class_names ) . '"><use xlink:href="' . esc_url( $href ) . '"></use></svg>';
 	}
+
+	/**
+	 * Gets image from assets.
+	 *
+	 * @since 3.15.0
+	 *
+	 * @param string $name        The name of the icon file at "assets/images/<name>.svg".
+	 *
+	 * @return string The SVG HTML.
```

Comment:
Should this docblock specify SVG? It actually looks like we're fetching pngs, so probably we should make the text here more general and not specify a type.

Reply:
Both these comments are actually, because of the c/p